docs/pages/concepts/logical/designs.md

```diff
@@ -14,9 +14,37 @@ Like a Google Doc, Designs are your primary tool for collaborative authorship of
 
 <!-- ### Using Designs -->
 
+A Design consists of **Components** and/or **Patterns**. A Design is the deployable unit in Meshery. Designs are how the users can describe the desired infrastructure state.
+There cannot be two components with the same name within a Design. However, there can be two components with the same name in different Designs.
+
 ### Design Relationships and Restrictions
 
-- Access to Designs may be granted to one or more users.
+- Designs belong to one an only one Workspace at any given time. Designs may be transferred between Workspaces.
```

Comment:
Designs belong to only one Workspace at any given time. Designs may be transferred between Workspaces.

Reply:
@leecalcote, please suggest if "can" or "may" is appropriate for the list of Designs.

Reply:
THank you. Suggestion accepted.

Reply:
You're right that "can" is the more appropriate term here.